admin/services/check-complete.service.js

```diff
@@ -45,7 +46,8 @@ checkCompleteService.completeCheck = async function (completedCheck) {
   if (config.autoMark) {
     // HACK temporary way to mark checks until we move to a dedicated scheduled process
     const check = await completedCheckDataService.sqlFindOneByCheckCode(completedCheck.data.pupil.checkCode)
-    await markingService.mark(check)
+    const checkForm = await checkFormDataService.sqlFindOneById(completedCheck.checkForm_id)
```

Comment:
Potentially this `checkForm` may not be found.  We wouldn't want to issue an error because of a marking problem, so perhaps it's best if wrap these two lines in a try/catch and swallow any errors.  We should only mark the check if there is a `checkForm`